core/primitives/src/version.rs

```diff
@@ -152,20 +152,46 @@ pub enum ProtocolFeature {
 
 /// Both, outgoing and incoming tcp connections to peers, will be rejected if `peer's`
 /// protocol version is lower than this.
-pub const PEER_MIN_ALLOWED_PROTOCOL_VERSION: ProtocolVersion = MAIN_NET_PROTOCOL_VERSION - 2;
+pub const PEER_MIN_ALLOWED_PROTOCOL_VERSION: ProtocolVersion = STABLE_PROTOCOL_VERSION - 2;
 
-/// Current protocol version used on the main net.
+/// Current protocol version used on the mainnet.
 /// Some features (e. g. FixStorageUsage) require that there is at least one epoch with exactly
 /// the corresponding version
-const MAIN_NET_PROTOCOL_VERSION: ProtocolVersion = 52;
+const STABLE_PROTOCOL_VERSION: ProtocolVersion = 52;
 
 /// Version used by this binary.
 #[cfg(not(feature = "nightly_protocol"))]
-pub const PROTOCOL_VERSION: ProtocolVersion = MAIN_NET_PROTOCOL_VERSION;
+pub const PROTOCOL_VERSION: ProtocolVersion = STABLE_PROTOCOL_VERSION;
 /// Current latest nightly version of the protocol.
 #[cfg(feature = "nightly_protocol")]
 pub const PROTOCOL_VERSION: ProtocolVersion = 126;
 
+/// The moment in time after which the voting for the latest client version should start on the
+/// mainnet version of the protocol.
+#[allow(dead_code)]
+const STABLE_PROTOCOL_UPGRADE_VOTING_START: Option<ProtocolUpgradeVotingSchedule> = None;
```

Comment:
so before each release, we should set some value here, right? 

(if yes - we should add more comments).

(and also I like the idea of this being more 'explicit' - by using the map from protocol version to the expected upgrade_voting_start)

Reply:
> so before each release, we should set some value here, right?
Yes, if we want the protocol upgrade to be delayed.

Implemented. I'm slightly concerned that this dictionary needs to be populated in the release branch, and thus will only contain at most one value.